cmake/FindHICMA-X.cmake

```diff
@@ -0,0 +1,108 @@
+
+# Copyright (c) 2017-2024 King Abdullah University of Science and Technology,
+# All rights reserved.
```

Comment:
Did you write this? Or getting it from another place?

Reply:
I mean the while file

Reply:
Yes, I wrote it